pt/_posts/2020-03-28-subsurface-mobile-3-0-released.MD

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title:  "Subsurface-mobile 3.0 released"
+lang:   "en"
```

Comment:
same here. The options are either not to add the post, to add the post in English but with the 'pt' language (that means it will be shown to people asking for Portuguese version, just in English, or to translate it...